src/app/pages/ErrorPage/ErrorPage.jsx

```diff
@@ -34,18 +39,22 @@ const ErrorMetadata = ({ dir, lang, messaging, brandName, themeColor }) => {
 };
 
 const ErrorPage = ({ errorCode }) => {
-  const { brandName, dir, lang, script, service, themeColor, translations } =
+  const { brandName, dir, lang, script, service, translations } =
     useContext(ServiceContext);
   const messaging = translations.error[errorCode] || translations.error[500];
 
+  const {
+    palette: { BRAND_BACKGROUND },
+  } = useTheme();
```

Comment:
This looks like a duplicate of lines 17-19 - is it possible to get the `BRAND_BACKGROUND` once, and use it in multiple places?

Reply:
Oops my bad, I didn't realise that about `useTheme` - I didn't see the whole file either! - ignore my comment!

Reply:
No worries, I think it makes more sense the way I have updated it.